File: Flip_A_Coin/flip_a_coin.py
import random
print("Here's your chance to win Huge prize")
print("All you have to do is guess what will be falling when we will be tossing the coin")
k=0
print("Your current balance=> ",k)
print("Choose H for Heads and T for Tails.")
c=input()
while(True):
    s=100
    x=random.randint(1,2)
    if x==1 and (c=='H' or c=='h'):
        print("You won ",s,"$")
        k=k+s
    elif  x==2 and (c=='T' or c=='t'):
        print("You won ",s,"$")
        k=k+s
    else:
        print("You Lost all your Monney..... :(")
        break

    print("Do you want to walk away with your money or  you want to keep playing Y/N")
    c2=input()
    if c2=='N' or c2=='n':
        print("Wise men and women often take a step back. Good Choice. Your current Balance=> ",k)
        break
    print("Your current balance=> ",k)
    print("Choose H for Heads and T for Tails.")
    c=input()
    s=s+100

File: Flip_A_Coin/FlipACoin.java
import java.util.*;
public class FlipACoin
{
	public static void main(String args[])
	{
		Scanner sc=new Scanner(System.in);
		List<Integer> list = new ArrayList<>();
		list.add(0);
        list.add(1);
		FlipACoin obj=new FlipACoin();
		System.out.println("Here's your chance to win Huge prize");
		System.out.println("All you have to do is guess what will be falling when we will be tossing the coin");
		System.out.println("Choose H for Head and T for Tails");
		char ch=sc.next().charAt(0);
		int s=0;
		while(true)
		{
			int x=obj.getRandomElement(list);
			if ((x==0) && (ch=='H' || ch=='h'))
			{
				s=s+100;
				System.out.println("You Won 100"+"$");
			}
			else if ((x==1) && (ch=='T' || ch=='t'))
			{
				s=s+100;
				System.out.println("You Won 100"+"$");
			}
			else
			{
				System.out.println("You Lost All Your Money :(");
				break;
			}
			System.out.println("Do You Still want to keep playing? Y/N");
			char z=sc.next().charAt(0);
			if (z=='N'||z=='n')
			{
				System.out.println("Wise Men and Women of take a step back wise choice\nYour Current Balance=> "+s);
				break;
			}
			System.out.println("Choose H for Head and T for Tails");
			ch=sc.next().charAt(0);
		}
	}
	public int getRandomElement(List<Integer> list)
    {
        Random rand = new Random();
        return list.get(rand.nextInt(list.size()));
    }
}